src/VkBootstrap.h

```diff
@@ -844,6 +845,27 @@ class SwapchainBuilder {
 	// Set the number of views in for multiview/stereo surface
 	SwapchainBuilder& set_image_array_layer_count(uint32_t array_layer_count);
 
+	// Convenient named constants for passing to set_desired_min_image_count().
+	// Note that it is not an `enum class`, so its constants can be passed as an integer value without casting
+	// In other words, these might as well be `static const int`, but they benefit from being grouped together this way.
```

Comment:
Well, this answers why use enum and not enum class. Though, with that comment I kinda wonder if they should be static const int, so that any code cleanliness warnings don't get issued for not using enum class.

Your call, I'm fine with either way.